server/import.go

```diff
@@ -32,7 +32,33 @@ func CreateImportBlob(ekPub crypto.PublicKey, sensitive []byte, pcrs *tpmpb.Pcrs
 	private := createPrivate(sensitive, ek.NameAlg)
 	public := createPublic(private, ek.NameAlg, pcrs)
 
+	return createImportBlobHelper(ek, public, private, pcrs)
+}
+
+// CreateSigningKeyImportBlob uses the provided public EK to encrypt the signing
+// key into import blob format. The returned import blob can be used to import
+// the signing key into the TPM associated with the provided EK without exposing
+// the private area to the TPM's OS. A non-nil pcrs parameter adds a requirement
+// that the TPM must have specific PCR values to use the signing key.
+func CreateSigningKeyImportBlob(ekPub crypto.PublicKey, signingKey crypto.PrivateKey, pcrs *tpmpb.Pcrs) (*tpmpb.ImportBlob, error) {
+	ek, err := CreateEKPublicAreaFromKey(ekPub)
+	if err != nil {
+		return nil, err
+	}
+	rsaKey, ok := signingKey.(*rsa.PrivateKey)
+	if !ok {
+		return nil, fmt.Errorf("unsupported signing key type: %T", signingKey)
+	}
+	private, public, err := createPrivatePublicRSA(rsaKey, ek.NameAlg, pcrs)
```

Comment:
I think we can avoid passing the NameAlg around in this function completely. It can't be changed by the user, so we can just do:

`var defaultNameAlg = tpm2tools.DefaultEKTemplateRSA().NameAlg`

and just use `defaultNameAlg` wherever we need it.